src/retrace/plugin_class.py

```diff
@@ -0,0 +1,26 @@
+#Singleton class for saving path to the plugin directory
```

Comment:
Oh no, move this file to the retrace directory. Rename the class to "Plugins". Add method "all()" which will the list of all plugins. Add code that will use the new "Plugins" class to `plugins/__init__.py`.